src/app-directory/specification/appd.yaml

```diff
@@ -447,12 +447,57 @@ components:
         lang:
           type: string
           pattern: '^[a-z]{2}(-[a-zA-Z0-9]{2,8}){0,1}$'
-          description:  a language tag that specifies the primary language of both application and its AppD entry, as defined by IETF RFC 5646.
+          description: >- 
+            A language tag that specifies the primary language of both 
+            application and its AppD entry, as defined by IETF RFC 5646.
         description:
           type: string
           description: >-
             Description of the application. This will typically be a 1-2
-            paragraph style blurb about the application. Allow mark up language
+            paragraph style blurb about the application. 
+        categories:
+          description: >-
+            An array of string categories that describe the application. 
+            These are meant as a hint to catalogs or stores listing FDC3-enabled 
+            apps and it is expected that these will make a best effort to find 
+            appropriate categories (or category) under which to list the app. 
+            AppD record authors are encouraged to use lower-case and, where 
+            possible, to select categories from the following list: <br>
+            - allocations<br>
+            - analytics<br>
+            - charts<br>
+            - chat<br>
+            - communication<br>
+            - compliance<br>
+            - crm<br>
+            - developer tools<br>
+            - events<br>
+            - execution management<br>
+            - file sharing<br>
+            - market data<br>
+            - news<br>
+            - networking<br>
+            - order management<br>
+            - office apps<br>
+            - portfolio management<br>
+            - presentation<br>
+            - pricing<br>
+            - productivity<br>
+            - research<br>
+            - risk<br>
+            - screen sharing<br>
+            - security<br>
+            - spreadsheet<br>
+            - trade cost analysis<br>
+            - trading system<br>
+            - training<br>
+            - travel<br>
+            - video<br>
+            - visualisation<br>
+            - weather<br>
```

Comment:
I know it's a nitpick, but I'm just wondering if there's a better way to achieve newlines in yaml than <br> tags on every line. I'm not very familiar with yaml, but I did find this stackoverflow answer that seems relevant: https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines

Reply:
~You can break lines in openapi descriptions written in yaml, but they won't render in the output... There are two options for rendering newlines and that's `\n` or `<br>` (although I should probably have written `<br />`... Personally I don't have much of a preference.~

~In theory the github flavoured markdown should be working so this list should have rendered and not needed the line breaks... but it didn't.~

switching from a `>` to a `|` block and putting a newline before the list caused it to render! much cleaner (in both code and rendered) now.